docs/physics/tardisgamma/decayenergy.ipynb

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
<p>I think we should discuss branching ratios here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>